src/http_server.rs

```diff
@@ -56,9 +59,12 @@ impl HttpServer {
             }
         });
 
+        let users_router = UsersRouter::new();
+
         let mut mount = Mount::new();
         mount.mount("/", Static::new(Path::new("static")))
-             .mount("/services", router);
+             .mount("/services", router)
+             .mount("/users_admin", users_router);
```

Comment:
There are several [endpoints ](https://github.com/fxbox/users/blob/master/doc/API.md#api-endpoints) with a path already including `users` so I figured that `users_admin` would be a better fit